frontend/src/metabase/public/components/PublicNotFound.jsx

```diff
@@ -1,20 +1,15 @@
 import cx from "classnames";
 import { t } from "ttag";
 
+import { NoDataError } from "metabase/components/errors/NoDataError";
 import CS from "metabase/css/core/index.css";
-import QueryBuilderS from "metabase/css/query_builder.module.css";
 
 import EmbedFrame from "./EmbedFrame";
 
 const PublicNotFound = () => (
   <EmbedFrame className={CS.spread}>
     <div className={cx(CS.flex, CS.layoutCentered, CS.flexFull, CS.flexColumn)}>
-      <div
-        className={cx(
-          QueryBuilderS.QueryErrorImage,
-          QueryBuilderS.QueryErrorImageNoRows,
-        )}
-      />
+      <NoDataError mb="1rem" />
```

Comment:
`margin-bottom` is from https://github.com/metabase/metabase/blob/f7a7df8d5c49ea37091b20c3868d8c17560a131d/frontend/src/metabase/css/query_builder.module.css#L149-L152

To see this component visit `/public/any-path-that-does-not-exist`

![image](https://github.com/metabase/metabase/assets/1937582/62af4fb9-ca51-4d9a-9b1a-32f041c7dff1)